### PR TITLE
[2.x] Test on PHP 8.2

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,44 +22,47 @@ jobs:
           - '5.3.*'
           - '5.4.*'
           - '6.0.*'
-          - '6.1.*'
```
Comment (chapterjason) [inline .github/workflows/ci.yml:24]: Lowest support is 6.0 for 6.x, why skip 6.1 and 6.2?

As time passed by, also consider to add 6.4.

**Review by chapterjason — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -21,44 +22,47 @@ jobs:
...
+            symfony: '4.4.*'
+            dependency: 'lowest'
+          - php: '8.2'
+            symfony: '5.3.*'
+            dependency: 'lowest'
       fail-fast: false
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```
Comment (Chris53897) [inline .github/workflows/ci.yml:65]: Can be changed to actions/checkout@v4

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Chris53897) [reply .github/workflows/ci.yml:24]: Can we bump minimum supported versions to `^5.4 || ^6.3 || ^7.0` as LTS Version, and latest actual version?

**Comment by Chris53897** [comment]
@GrahamCampbell Could you please add tests for php 8.3 ?

**Comment by chapterjason** [comment]
Solved by #83

**Pull request closed without merging**